### ISSUE TITLE
Build fails when using a src directory

### ISSUE THREAD

**Issue by skivis** [serverless-stack/open-next#18]
When running `open-next build`, specifically when creating the middleware bundle, a `src` directory is never taken in to account.

https://github.com/serverless-stack/open-next/blob/1f57e3dbe6cdae1a101c9a75f148984ef5bd5aa0/src/build.ts#L187

Excerpt from next docs:

> The src directory is very common in many apps and Next.js supports it by default.

So in my case the following would work:

```ts
const src: string = buildOutput.output['src/middleware'].files["index.js"].data;
```

But obviously both should be supported :)

**Comment by AlexisWalravens** [comment]
Can confirm I just had the same issue.

```
Traced Next.js server files in: 2.289s
Created all serverless functions in: 1.006s
Collected static files (public/, static/, .next/static): 4.206ms
┌────────────────────────────┐
 │ Generating OpenNext bundle │
└────────────────────────────┘
Using v0.4.2
Bundling server function...
Bundling image optimization function...
Bundling middleware edge function...
file:///Users/alexis.walravens/.npm/_npx/a94ce00fc9d9ef90/node_modules/open-next/dist/build.js:146
    const src = buildOutput.output.middleware.files["index.js"].data;
                                              ^

TypeError: Cannot read properties of undefined (reading 'files')
    at createMiddlewareBundle (file:///Users/alexis.walravens/.npm/_npx/a94ce00fc9d9ef90/node_modules/open-next/dist/build.js:146:47)
    at build (file:///Users/alexis.walravens/.npm/_npx/a94ce00fc9d9ef90/node_modules/open-next/dist/build.js:23:5)
```

**Comment by fwang** [comment]
Thanks @skivis @AlexisWalravens.

Fixed in v0.5.2
